firmware/mw_dual_motor_torque_ctrl/src/dual_motor_torque_ctrl.c

```diff
@@ -568,6 +568,29 @@ void main(void)
 	// At the beginning, there are no errors
 	gErrors.all = 0;
 
+	// Enable the motors programmatically.
+	gMotorVars[HAL_MTR1].Flag_enableSys = true;  // enable the system in general
+	gMotorVars[HAL_MTR1].Flag_Run_Identify = true;  // enable the motor
+	gMotorVars[HAL_MTR2].Flag_enableSys = true;  // enable the system in general
+	gMotorVars[HAL_MTR2].Flag_Run_Identify = true;  // enable the motor
+
+	// Speedup the initialization time by a factor of 10.
+	gUserParams[HAL_MTR1].ctrlWaitTime[CTRL_State_OffLine] /= 10;
+	gUserParams[HAL_MTR2].ctrlWaitTime[CTRL_State_OffLine] /= 10;
```

Comment:
I think it would be better for readability to set an explicit value here.  What is the initial value?  I have 5s in mind but can't find it in the code right now.

Just one concern when the wait time is too short: The motors will oscillate a bit in the beginning, so it should be long enough to ensure that it motors came to rest.  I agree that the current value is probably longer than needed but something below one second sounds a bit tight for me.  Maybe something like 1-2 seconds?